projects/common/src/lib/elements/manage/manage.component.scss

```diff
@@ -37,3 +37,7 @@
     }
   }
 }
+
+#last-sync-text{
+  margin-top:30px;

```

Comment:
Instead of setting this explicitly, use our 'spread' class in the html (comments above)